src/components/network-modification-tree.jsx

```diff
@@ -80,20 +80,17 @@ const NetworkModificationTree = ({
 
     const nodesMap = useMemo(() => new Map(nodes.map((n) => [n.id, n])), [nodes]);
 
-    const updateNodePositions = useCallback(
-        (model) => {
-            if (model && model.treeNodes?.length > 0) {
-                setNodes(getTreeNodesWithUpdatedPositions(model.treeNodes));
-                setEdges([...model.treeEdges]);
-                window.requestAnimationFrame(() => fitView());
-            }
-        },
-        [fitView, setNodes, setEdges]
-    );
+    const updateNodePositions = useCallback(() => {
+        if (treeModel && treeModel.treeNodes?.length > 0) {
+            setNodes(getTreeNodesWithUpdatedPositions(treeModel.treeNodes));
+            setEdges([...treeModel.treeEdges]);
+            window.requestAnimationFrame(() => fitView());
```

Comment:
This line should be removed as it changes the screen position when interacting with nodes.